pbdf/Issues/surfnet-2/description.xml

```diff
@@ -29,20 +29,20 @@
-		<en>Add your education and research data to your IRMA app.</en>
-		<nl>Laad je onderwijs en onderzoek gegevens in je IRMA-app.</nl>
+		<en>Add your education and research data to your Yivi app.</en>
+		<nl>Laad je onderwijs en onderzoek gegevens in je Yivi-app.</nl>
 	</FAQIntro>
 	<FAQPurpose>
-		<en>You can use your education and research data in your IRMA app to prove that you are studying or working at a particular school, university or research institute.</en>
-		<nl>Met jouw onderwijs en onderzoek gegevens in de IRMA-app kun je bewijzen dat je studeert of werkt bij een bepaalde school, universiteit of onderzoeksinstelling.</nl>
+		<en>You can use your education and research data in your Yivi app to prove that you are studying or working at a particular school, university or research institute.</en>
+		<nl>Met jouw onderwijs en onderzoek gegevens in de Yivi-app kun je bewijzen dat je studeert of werkt bij een bepaalde school, universiteit of onderzoeksinstelling.</nl>
 	</FAQPurpose>
 	<FAQContent>
 		<en>You will add the following data:\n• first name,\n• last name,\n• e-mail address,\n• institution,\n• employee/student,\n• local registration number.</en>
 		<nl>Je laadt de volgende gegevens:\n• voornaam,\n• achternaam,\n• e-mailadres,\n• instelling,\n• medewerker/student,\n• lokaal registratienummer.</nl>
 	</FAQContent>
 	<FAQHowto>
-		<en>If you study or work at a participating education / research institution, you can add personal data from this institution to your IRMA app.\n\n Complete the following steps.\n1. Select your own institution within the SURFconext environment; if it is not listed, your institution is not participating and you cannot add this card.\n2. Log in at the website of your own institution, in the usual manner.\n3. Confirm that you wish to add your data to IRMA. \n\nYou now receive a card with your education / research details.</en>
-		<nl>Als je studeert of werkt bij een deelnemende onderwijs- / onderzoeks-sinstelling kun je persoonsgegevens van deze instelling toevoegen in je IRMA app.\n\nDoorloop de volgende stappen.\n1. Selecteer je eigen instelling in de SURFconext omgeving; als die instelling niet voorkomt in de lijst, doet die niet mee en kun je dit kaartje niet laden.\n2. Log in op website van je eigen instelling op gebruikelijke wijze.\n3. Kies voor het overzetten van je gegevens naar je IRMA app.\n\nJe ontvangt nu een kaartje met je onderwijs- / onderzoeks-gegevens.</nl>
+		<en>If you study or work at a participating education / research institution, you can add personal data from this institution to your Yivi app.\n\n Complete the following steps.\n1. Select your own institution within the SURFconext environment; if it is not listed, your institution is not participating and you cannot add this card.\n2. Log in at the website of your own institution, in the usual manner.\n3. Confirm that you wish to add your data to Yivi. \n\nYou now receive a card with your education / research details.</en>
```

Comment:
If you study or work at a participating educational or research institution, you can add your personal data known to this institution in your Yivi app.\n\nFollow the following steps:\n1. Select your own institution in the SURFconext environment. If the institution does not appear in the list, it is not included and you cannot load this card.\n2. Log in to your own institution's website in the usual way.\n3. Choose to transfer your data to your Yivi app.\n\nYou will now receive a card with your data known to the educational or research institution.